### DISCUSSION TITLE
Autocreate folder/folder-structure in users home-dir

### DISCUSSION THREAD

**Discussion by minimax75** [drakkan/sftpgo#990] [Q&A] [answered]
Hi,
is ist possible to define a folderstructure for the home-directory of an users. So that it will be automatically create as soon as a new user is added to sftpgo?

**Comment by drakkan** [comment, marked as answer]
Yes, try the eventmanager in the development version

**Reply by minimax75** [reply]
Hi drakkan,
I have tried your proposed approach and iit works.
Thank you
